satpy/readers/aapp_l1b.py

```diff
@@ -33,10 +33,22 @@
 import xarray as xr
 from dask import delayed
 
-from satpy import CHUNK_SIZE
 from satpy.readers.file_handlers import BaseFileHandler
+from satpy.utils import get_chunk_pixel_size
+
+CHANNEL_DTYPE = float
```

Comment:
Why `float` versus the more explicit `np.float64`? Mostly just curious.

Reply:
My understanding was that using plain python type was prefered, but I could be wrong :)

Reply:
I can't see anything about my previous comment anywhere, so I'll just make this explicit.

Reply:
```suggestion
CHANNEL_DTYPE = np.float64
```

Reply:
Using the plain python type is preferred over the deprecated `np.float`, but I'm not sure `float` is preferred over other numpy types.